content/_components/onedrive/index.md

```diff
@@ -6,152 +6,158 @@ description: This is the component for working with Microsoft OneDrive storage s
 
-![Get New And Updated Files Polling](img/get-new-and-update.png)
+#### Output Metadata
 
-#### List of Expected Config fields
+Depends on `Enable File Attachments` and `Emit behavior` fields.
+ * If `Emit behavior` field is equal to `Emit page` - object with property `results` that contains array of files
+ * If `Emit behavior` field is equal to `Emit individually`, file information will fulfill whole message
+ * If `Enable File Attachments` checked, for each file there will be additional field - `attachmentUrl`
 
-* **Drive Identity** - OneDrive instance to work with.
-* **Folder path** - Dropdown with folders to poll files from.
-* **Emit Behavior** -  Available options: default is `Emit Individually` emits each object in separate message, `Fetch All` emits all objects in one message.
-* **Start Time** - Start datetime of polling. Default min date: `-271821-04-20T00:00:00.000Z`.
-* **End Time** - End datetime of polling. Default max date: `+275760-09-13T00:00:00.000Z`.
-* **Size Of Polling Page** - Indicates the size of pages to be fetched. Defaults to `1000`.
-* **Expand Children** - Checkbox, trigger retrieves files from subfolders of chosen path, if enabled. Disabled by default.
-* **Enable File Attachments** - Checkbox for attaching files content to response. Disabled by default.
 
-## Actions
-
-### Get File
+#### Limitations
 
-Action to get item from OneDrive by provided path in selected disc.
+* OneDrive API doesn't support filtering - as result we collect information about all files from selected folders and filter them locally (inside component) for each trigger execution
+* From point above, option `Emit page` not always emit records according to `Page Size`
+* `Include subfolders` increase amount of API calls - additional call for each subfolder and inner folders
 
-![Get File](img/get-file.png)
-
-#### Input fields description
+## Actions
 
-* **Drive Identity** - OneDrive instance to work with. Selects by owner
-* **Enable File Attachments** - checkbox for attaching files content to action response
+### Create Folder
 
-#### Metadata fields description
+Create new folder in provided `path`. If `path` not exist component will fail.
 
-* **Path** - Full path to item to create or replace
+#### Configuration Fields
 
-#### Input example:
+* **Drive Identity** - (dropdown, required): OneDrive instance to work with
+* **Conflict Behavior** - (dropdown, optional, `Fail` by default) - Select one of options to handle case when folder already exists:
+  * **Fail** - Fails if folder with same name already exists under provided `path`
+  * **Generate new name** - If folder with same name already exists under provided `path` then will be created new folder with different name. Examples: `folder_name` (already exist) -> `folder_name 1` (will be created), `folder_name 1` (already exist) -> `folder_name 1 1` (will be created)
+  * **Upsert** - If folder already exist, you will get information about this folder
```

Comment:
* **Upsert** - If the folder already exists, you will get information about this folder